File: csharp/Concepts/Conditionals.cs
namespace ProgrammingFundamentals.Concepts
{
    public class Conditionals
    {
        public static void Demonstrate()
        {
            Console.WriteLine("\n=== Conditional Statements ===");
            int number = 15;

            if (number > 0)
            {
                Console.WriteLine($"{number} is positive");
            }
            else if (number < 0)
            {
                Console.WriteLine($"{number} is negative");
            }
            else
            {
                Console.WriteLine("The number is zero");
            }
        }
    }
}

File: java/concepts/Conditionals.java
package concepts;

public class Conditionals {
    public static void demonstrate() {
        System.out.println("\n=== Conditional Statements ===");
        int number = 15;

        if (number > 0) {
            System.out.println(number + " is positive");
        } else if (number < 0) {
            System.out.println(number + " is negative");
        } else {
            System.out.println("The number is zero");
        }
    }
}
